rustls/src/webpki/mod.rs

```diff
@@ -104,13 +107,13 @@ fn crl_error(e: webpki::Error) -> CertRevocationListError {
     }
 }
 
-fn borrow_crls(
-    crls: &Vec<webpki::OwnedCertRevocationList>,
-) -> Vec<&dyn webpki::CertRevocationList> {
-    #[allow(trivial_casts)] // Cast to &dyn trait is required.
+fn parse_crls(
+    crls: Vec<CertificateRevocationListDer>,
+) -> Result<Vec<CertRevocationList>, CertRevocationListError> {
```

Comment:
Let's make the `CertRevocationList` lifetime explicit here? (And the one for `CertificateRevocationListDer`, I suppose.)

Reply:
Fixed!